code/game/objects/items/ego_weapons/teth.dm

```diff
@@ -149,3 +149,14 @@
 		return
 	else
 		return ..()
+
+/obj/item/ego_weapon/cute
+	name = "SO CUTE!!!"
+	desc = "One may think, 'How can a weapon drawn from such a cute Abnormality be any good?' \
+		However, the claws are actually quite durable and sharp."
+	icon_state = "cute"
+	force = 13
+	attack_speed = 0.5
+	damtype = RED_DAMAGE
+	armortype = RED_DAMAGE
```

Comment:
Red damage is the default, so this is obsolete.

Reply:
All ego weapons (I didn't check guns though) still puts it in even if they were red
Honestly at this point it's more like a theme
I'll erase it if you still don't like it though